timeseer/podman/roles/timeseer_data_service/tasks/main.yml

```diff
@@ -15,27 +15,32 @@
     timeseer_data_service_storage_mount: "{{ timeseer_data_service_dir }}"
   when: timeseer_data_service_podman_volume | default('') | trim == '' and timeseer_data_service_enable
 
-- name: Run ts-data-service as a container
-  containers.podman.podman_container:
-    name: timeseer-data-service
-    image: "{{ timeseer_data_service_image }}"
-    command: "/usr/bin/timeseer-data-service"
-    state: started
-    pull: "{{ timeseer_data_service_image_pull }}"
-    restart_policy: unless-stopped
-    network: "{{ timeseer_data_service_network }}"
-    volumes: "{{ volume_list }}"
-    log_driver: journald
-    log_options:
-      tag: "{{ timeseer_data_service_journal_tag }}"
-    env:
-      RUST_LOG: info
-    tty: true
-    ports: "{{ timeseer_data_service_ports }}"
+- name: Define timeseer-data-service volume list
+  ansible.builtin.set_fact:
+    data_service_volume_list: "{{ ds_base_volumes + (ds_custom_toml_volume if timeseer_data_service_config_dir | default('') | trim != '' else []) }}"
   vars:
-    base_volumes:
+    ds_base_volumes:
       - "{{ timeseer_data_service_storage_mount }}:/usr/src/app/data-service"
-    custom_toml_volume:
+    ds_custom_toml_volume:
       - "{{ timeseer_data_service_config_dir }}:/usr/src/app/Timeseer-data-service.toml"
-    volume_list: "{{ base_volumes + (custom_toml_volume if timeseer_data_service_config_dir | default('') | trim != '' else []) }}"
+
+- name: Prepend 'systemd-' to network
```

Comment:
Why is this not required for the network in the `timeseer` role?

Reply:
likely because the network configuration is already being handled within the quadlet file itself

`[Network]
Label={{ timeseer_network }}`


This specifies the network for the container directly in the configuration file. By doing so, the network is created and labeled correctly without needing additional manipulation in the playbook.


If you add the [Network] section to multiple quadlet files, it can lead to the creation of multiple networks i guess

Reply:
"likely", "guess" seems to indicate that the issue is not fully understood.

Can you point to the podman documentation where they explain that for the second container in a network, the network should be prefixed?

Reply:
Shouldn't `Network=timeseer.network` be used in the quadlet file as is done in the Timeseer container? Alternatively, the name of the `.network` file could be templated as well.